tests/localjuno/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     image: juno
     build:
       context: ../../
-      dockerfile: Dockerfile
+      dockerfile: tests/localjuno/init.Dockerfile
```

Comment:
What is the difference between init.Dockerfile here & the parent folder Dockerfile?
My idea is that we delete the init.Dockerfile / old docker work or merge it together all into localjuno (I think it is just a gas change param)

Reply:
Yea I agree. may want to just rename it from init.Dockerfile -> like e2e.Dockerfile or something then so it better describes it if we are keeping all 3.

Then in the future we can narrow it down to the best :)